ctrl.c

```diff
@@ -838,6 +841,36 @@ static struct pv_cmd* pv_ctrl_process_endpoint_and_reply(int req_fd,
 			pv_ctrl_write_ok_response(req_fd);
 		} else
 			goto err_me;
+	} else if (pv_str_startswith(ENDPOINT_DRIVERS, strlen(ENDPOINT_DRIVERS), path)) {
+		driverkey = pv_ctrl_get_file_name(path, sizeof(ENDPOINT_DRIVERS), path_len);
+
+		if (!driverkey) {
+			pv_log(WARN, "HTTP request has bad driver alias");
+			pv_ctrl_write_error_response(req_fd, HTTP_STATUS_BAD_REQ, "Request has bad driver key name");
+			goto out;
+		}
+
+		if (!strncmp("PUT", method, method_len)) {
```

Comment:
I don't know if I like using PUT for this one.. this kind of unload/upload ops seems more suited to a POST similar to the one we use for commands, and then put the op and driver in a json in the body like "{"op":"LOCAL_RUN","payload":"4"}". Not very strong on this one, but I'd do it like that to keep consistency.

Reply:
lets keep it at PUT for now.... there is not clear winner, in REST world a PUT would about an endpoint taht updates an existin resource, while a post would be about an endpoint where you can create new documents/resources. I think module loading is like activation and deactivation that REST APIs usually do with a PUT unless its about a activate/deactivate job resource that gets POSTED and then processed.